src/index.html

```diff
@@ -21,7 +21,322 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a
+        href="index.html"
+        class="moyo-logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="moyo-logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <a
+              href="index.html"
+              class="nav__list-link is-active"
+            >
+              apple
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="index.html"
+              class="nav__list-link"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="index.html"
+              class="nav__list-link"
+            >
+              smartphones
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="index.html"
+              class="nav__list-link"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="index.html"
+              class="nav__list-link"
+            >
+              gadgets
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="index.html"
+              class="nav__list-link"
+            >
+              tablets
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="index.html"
+              class="nav__list-link"
+            >
+              photo
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="index.html"
+              class="nav__list-link"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__picture"></div>
+
+        <div class="card__name">
```

Comment:
```suggestion
        <h3 class="card__name">
```
Consider using semantic elements